mobile/src/main/java/de/michelinside/glucodatahandler/widget/GlucoseBaseWidget.kt

```diff
@@ -241,6 +245,37 @@ abstract class GlucoseBaseWidget(private val type: WidgetType,
             else
                 remoteViews.setViewVisibility(R.id.cobText, View.VISIBLE)
         }
+
+
+        if (hasBatteryLevel) {
+            var deviceNameStr = "No WearOS Device"
+            var levelStr = "?%"
+            var levelColour = Color.RED;
+
+            if (WearPhoneConnection.nodesConnected) {
+                if (WearPhoneConnection.connectionError) {
+                    deviceNameStr = "Connection error"
+                }
+                else {
+                    WearPhoneConnection.getNodeBatterLevels().firstNotNullOf { (name, level) ->
```

Comment:
I also thought about supporting for more than one watch, but this is more a corner case and would make it too complicated... So using only the first is fine!

Reply:
I did think about this too.. I figured it would be unlikely a user would have more than one device